src/DateTimeFormatter.php

```diff
@@ -45,6 +57,20 @@ public function setFormat($format)
         return $this;
     }
 
+    /**
+     * Set whether or not to throw an exception in the event that DateTime
+     * cannot parse the input value
+     *
+     * @param  bool $throwException
+     * @return self
+     */
+    public function setThrowInvalidDateException(bool $throwException): self
+    {
+        $this->throwInvalidDateException = $throwInvalidDateException;
```

Comment:
`$throwException` vs. `$throwInvalidDateException`

See also: https://travis-ci.org/zendframework/zend-filter/jobs/506412742#L629

Reply:
Strange that I don't get these Es when I run tests.